packages/amplication-client/src/VersionControl/useBuildWatchStatus.tsx

```diff
@@ -41,6 +41,19 @@ const useBuildWatchStatus = (build: models.Build) => {
 
 export default useBuildWatchStatus;
 
+function shouldReload(build: models.Build | undefined): boolean {
+  return (
+    !build ||
+    build.status === models.EnumBuildStatus.Running ||
+    Boolean(
+      build.deployments?.some(
+        (deployment) =>
+          deployment.status === models.EnumDeploymentStatus.Waiting
+      )
+    )
```

Comment:
I think ending || false will be better here than wrapping with Boolean as some reruns Boolean value